lib/dolly/property.rb

```diff
@@ -2,7 +2,7 @@ module Dolly
   class Property
     attr_writer :value
     attr_accessor :name
-    attr_reader :class_name, :default
+    attr_reader :class_name, :default, :subproperties
```

Comment:
@neojin their sub-properties are their own instances, and are allowed to have their own sub-properties :)